test/compute/2019-12-01/armcompute/zz_generated_models.go

```diff
@@ -268,12 +268,6 @@ type BootDiagnosticsInstanceView struct {
 	Status *InstanceViewStatus `json:"status,omitempty" azure:"ro"`
 }
 
-// CloudError - An error response from the Compute service.
-type CloudError struct {
```

Comment:
Take this file as an example: we are removing:
- errors
- models that are not referenced in the swagger (I know that they have one in gallery `GalleryArtifactSource` and all its siblings)

But I think we should have more to omit in this RP, such like, `Resource` or `TrackedResource` and other kind of `allOf` models. They are never directly used by our generated code, and if they switch to the base type in common-types, and when they do that, there usually some slightly name difference - we will have troubles.
Like when they are defining swaggers by themselves, for simplification consideration, they would just name an entity with id, name, type, systemData, location and tag as a `Resource`, but in fact, in common-types, this thing is called a `TrackedResource` or `ProxyResource`. When they switch to this, we will see the Resource is removed and we have a TrackedResource, both of them are usually never referenced in our SDK

Reply:
I add the constrains for the parents. If the type is not a discriminator, it will not be iterated.